CONTRIBUTING.md

```diff
@@ -65,6 +66,13 @@ it here:
 [hack/git_client_side_hooks/pre-commit](hack/git_client_side_hooks/pre-commit).
 You can run `make install-hooks` to copy it to your local `.git/hooks/` folder, and remove it via `make uninstall-hooks`
 
+### Pre-Push Hook
```

Comment:
I think you can use `Git Client Hooks` as the title to merge `Pre-Commit` and `Pre-Push` hooks.